specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -2375,14 +2402,14 @@
     "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/providers/Microsoft.Billing/billingRoleDefinitions/{billingRoleDefinitionName}": {
       "get": {
         "tags": [
-          "billingRoleDefinition"
+          "BillingRoleDefinition"
         ],
         "x-ms-examples": {
           "BillingAccountRoleDefinition": {
             "$ref": "./examples/BillingAccountRoleDefinition.json"
           }
         },
-        "operationId": "BillingAccountBillingRoleDefinition_Get",
+        "operationId": "BillingRoleDefinition_GetByBillingAccountName",
```

Comment:
BillingRoleDefinition**s**_GetByBillingAccountName

Make plural in other APIs as well.